samples/ngIdentity/README.md

```diff
@@ -0,0 +1,114 @@
+
+## Run the sample
+
+Either run or inspect the launch settings for your projects. "Client" refers to the `ngIdentity.client` project and "server" refers to the `ngIdentity.server` host project. 
+
+* Modify `proxy.conf.js` and ensure the URL and port are correct for your server project. 
+* Modify `home.component.html` to post to the server URL.
+* Run the project
+
+> Sometimes a race condition can occur where the Angular build lags behind the server. The server will automatically kick off an Angular build if it can't connect to the client, so this results in two instances of the Angular build and the second will fail with a port conflict. When this happens, simply close the _first_ Angular client to launch and the second will succeed. If you close the second, it will end the debug session.
+
+## Understand the sample
+
+The identity APIs were added to the server project, then the client was modified to use the APIs.
+
+### Server
+
+The following changes were made to the server project to support authentication and authorization:
+
+1. Added the `Microsoft.AspNetCore.Identity.EntityFrameworkCore` NuGet package.
+1. Added the `Microsoft.EntityFrameworkCore.InMemory` NuGet package. You can use any relational EF Core provider, but you will need to properly create/migrate the database (see [Migrations Overview](https://learn.microsoft.com/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli)).
+1. Added the identity user and database.
+   ```csharp
+   class MyUser : IdentityUser { }
+
+   class AppDbContext(DbContextOptions<AppDbContext> options) :
+      IdentityDbContext<MyUser>(options)
+   {    
+   } 
+   ```
+1. Configured the identity services. The code here changes the default behavior of returning a `404` when not authenticated to a `401` so the client can automatically redirect to the login page.
+   ```csharp
+   builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme)
+    .AddIdentityCookies()
+    .ApplicationCookie!.Configure(opt => opt.Events = new CookieAuthenticationEvents()
+    {
+        OnRedirectToLogin = ctx =>
+        {
+            ctx.Response.StatusCode = 401;
+            return Task.CompletedTask;
+        }
+    }); 
+    builder.Services.AddAuthorizationBuilder();
+   ```
+1. Configured the EF Core `DbContext` to use the in-memory identity database, and turned on the identity APIs.
+   ```csharp
+   builder.Services.AddDbContext<AppDbContext>(
+      options => options.UseInMemoryDatabase("AppDb"));
+
+   builder.Services.AddIdentityCore<MyUser>()
+     .AddEntityFrameworkStores<AppDbContext>()
+     .AddApiEndpoints();
+   ```
+1. Mapped the identity API endpoints: `app.MapIdentityApi<MyUser>();`
+1. Added a logout method. This isn't included in the endpoints because it may change based on your configuration (i.e. cookies, tokens, both). More on this method below.
```

Comment:
```suggestion
1. Added a `logout` method. The `logout` method isn't included in the endpoints because it may change based on the configuration, i.e. cookies, tokens, or both. <!--More on this method below. We typically don't use below in the docs. In the following if that's appropriate, or later in the article, but I'm guessing we really don't need it.-->
```